data/1.0/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Jakarta Data 1.0 (under development)"
-date: 2023-03-04
-summary: "Under Development"
+date: 2023-05-30
+summary: "Release supporing Jakarta EE 11"
```

Comment:
Minor typo.  "supporing" should be "supporting"

Reply:
Thank you Natan.